### PR TITLE
Add LICENSE, update README.md instructions

### PR DESCRIPTION
**Pull request by rheaditi**
* Add MIT License
* Update `README.md` for set-up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,78 @@
...
-### Usage
+## Usage
 
-pivotal-flow contains two commands to automate your workflow
+`pivotal-flow` has two commands to automate your workflow:
 
-1. `pivotal-flow-start`
-2. `pivotal-flow-check`
+1. `pivotal-flow-start`:  
+  To set up hooks for Setting up your environment to create new stories via the command line.
```
Comment (rajanand02) [inline README.md]: hooks? and `S` is in capital

File: README.md [hunk 2]
```diff
@@ -1,37 +1,78 @@
...
 
-### References
+You can add the `pivotal-flow-check` as a post-checkout hook via [`husky` 🐶][husky].
+Just add it to your `package.json` file, for example:
 
-1. [Git tracker](https://github.com/stevenharman/git_tracker)
-2. [Git hooks](https://git-scm.com/docs/githooks#_post_checkout)
-3. [Husky](https://github.com/typicode/husky)
+```diff
+diff --git a/package.json b/package.json
```
Comment (rajanand02) [inline README.md]: do we need diff snippet here?

Comment (rheaditi) [reply README.md]: faster download than an image.. :P

File: README.md (hunk 1 above)
Comment (rheaditi) [reply README.md]: nope! my bad. 😞

**Pull request merged**